vpn-router-web/src/main/bundles/README.md

```diff
@@ -1,38 +0,0 @@
-This directory is automatically generated by Vaadin and contains the pre-compiled
```

Comment:
I didn't have a goal to specifically delete bundles but I've thought that maybe it's a good idea to run vaadinClean task before pushing a branch) because we don't have any custom bundles that need to be included to the project. it is said that this task clean the project and removes generated folders
<img width="1079" alt="Screenshot 2024-09-08 at 15 23 31" src="https://github.com/user-attachments/assets/cd98995f-faa8-48e2-ab68-57a4b4f943a5">
. what do you think about it?

Reply:
Now I've returned the bundles folder back

Reply:
Why did you return it?))

Reply:
There are two reasons:
1. README says that this folder should be added to version control system)
2. [documentation](https://vaadin.com/docs/v23/guide/start/gradle) says that "You need to run this (vaadinClean) task if you upgrade the Vaadin version, and in other similar situations." And it seems we don't need to run this task before pushing to the repository